pkgs/applications/virtualization/gvisor/default.nix

```diff
@@ -18,11 +18,11 @@ buildGoModule rec {
   src = fetchFromGitHub {
     owner = "google";
     repo = "gvisor";
-    rev = "bf8eeee3a9eb966bc72c773da060a3c8bb73b8ff";
-    sha256 = "sha256-rADQsJ+AnBVlfQURGJl1xR6Ad5NyRWSrBSpOFMRld+o=";
+    rev = "cdaf5c462c4040ed4cc88989e43f7d373acb9d24";
```

Comment:
commit doesn't match https://github.com/google/gvisor/releases/tag/release-20231113.0
also, maybe it should be `rev = "release-${version}"`

Reply:
gvisor tracks synthetic branch [`go`](https://github.com/google/gvisor/commits/go) because the release branches don't work with the normal go toolchain.

That commit is latest "Merge release-20231113.0-* (automated)" on the `go` branch.

Reply:
yes, thank you for explanation. I can see comment now 

```
that can be used to build gvisor without bazel.
```